src/components/instance-form-builder/InstanceFormBuilder.jsx

```diff
@@ -68,7 +68,11 @@ class InstanceFormBuilder extends React.Component {
                     contents={i18n.t("All your changes will be lost. Are you sure?")}
                 />
 
-                <FormHeading title={title} onBackClick={this.cancelSave} />
+                <PageHeader
+                    title={title}
+                    onBackClick={this.cancelSave}
+                    helpText={i18n.t("Help text")}
```

Comment:
If we don't have a proper help text now we could just remove it for now and define it later on. @adrianq Might have more insight about this.

Reply:
We'll wait a bit on this, if not I'll try and write up something coherent

Reply:
This is fine for now. We can always revisit it at a later stage. I just want to show the client what we can potentially have and if it makes sense for them